Automattic/wp-calypso
# Backup: Add constants for 3TB and 5TB add-on storage

This PR adds the necessary constant references, translations, etc. to enable 3TB and 5TB add-on storage products for Jetpack Backup.

Related to D101847-code, https://github.com/Automattic/wpcomsh/pull/1261.
Partially resolves peaFOp-Me-p2, 1202858161751496-as-1203947014200710.

## Proposed Changes

* Add product slugs for 3TB and 5TB add-on storage (in both monthly and yearly billing terms, though only monthly is currently available).
* Associate product slugs so that they're recognized as part of the collection of add-on storage products.
* Add friendly translated names and descriptions for each new product, and associate them with their corresponding product ID in the Store.

## Testing Instructions

It's hard to test this PR without its other co-dependent PRs applied as well, in one cohesive environment. Steps you can take:

* Verify all unit and end-to-end tests still pass.
* Evaluate code to ensure new additions are correct -- lots of content was quickly copied, pasted, and modified.
* If you're able, add one of the new products to an existing site you own and check that it's displayed correctly in the usual places (e.g., My Plan, Purchases, etc.)
* If you're able, add one of the new products to an existing site you own and check that the proper storage limit is shown on the Backup page.

## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203947014200710
